File: 151004/Ermolovich/DC_Lab1/DC_Lab1/DC_Lab1/DTO/LabelRequestTo.cs
using DC_Lab1.DTO.Interface;

namespace DC_Lab1.DTO
{
    public class LabelRequestTo(int? Id, string? Name) : IRequestTo
    {
        public int? Id { get; set; } = Id;

        public string? name { get; set; } = Name;
    }
}

File: 151001/Verghel/DC5/publicator/src/main/java/by/bsuir/dto/LabelRequestTo.java
package by.bsuir.dto;

import jakarta.validation.constraints.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Set;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class LabelRequestTo {
    private Long id;
    @NotBlank
    @Size(min = 2, max = 32)
    private String name;
}
